web/package.json

```diff
@@ -5,48 +5,35 @@
   "license": "MIT",
   "scripts": {
     "data": "cd .. && python3 scripts/convert_to_web_app_json.py",
-    "clear": "node ./lib/clear.js",
     "dev": "nodemon --config config/nodemon/dev.json",
     "dev:start": "yarn install && yarn dev:nowatch || cd .",
-    "dev:nowatch": "cross-env NODE_ENV=development BABEL_ENV=development NODE_OPTIONS=--max_old_space_size=5000  babel-node --config-file \"./babel-node.config.js\" --extensions \".js,.ts\" -- node_modules/.bin/next dev --hostname 0.0.0.0 --port 3000",
-    "dev:start_win": "yarn install && yarn dev:nowatch_win || cd .",
-    "dev:nowatch_win": "cross-env NODE_ENV=development BABEL_ENV=development NODE_OPTIONS=--max_old_space_size=5000  babel-node --config-file \"./babel-node.config.js\" --extensions \".js,.ts\" -- ./node_modules/next/dist/bin/next dev --hostname 0.0.0.0 --port 3000",
-    "dev:clean": "rimraf '.build/development/{*,.*}' 'node_modules/.cache'",
-    "prod:build": "yarn run next:build && yarn run next:export",
-    "prod:build:ci": "yarn run prod:build",
+    "dev:nowatch": "cross-env NODE_ENV=development BABEL_ENV=development next dev --port 3000",
+    "dev:clean": "rimraf '.build/development/**/*' 'node_modules/.cache'",
```

Comment:
You deleted `prod:clean`. Is this anymore useful in this state?
* Should we delete it, too?
* Should we adapt it to delete `build`, `playwright-report`, ... (i.e. everything that is autogenerated somehow)?

Reply:
I just moved the `prod:clean` to a different line but you are right, as the `watch` scripts are gone it is not needed anymore. Overall, caches and autogenerated things are maintained by the tools that create them, so we really don't need a script to clean them every time. I will remove both `clean` scripts.